contracts/Fitup.sol

```diff
@@ -5,14 +5,21 @@ contract Fitup {
     
     address owner;
 
+    struct Ngo {
+        string name;
+        address organisation;
+    }
+
     struct Bet {
         uint256 amount;
         address organisation; 
     }
```

Comment:
Bet, the structure will need to include other variables that we agreed on.

Reply:
You are right, can you add this when you create the createBet function?